drivers/sensor/hx711/hx711.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2020 George Gkinis
+ * Copyright (c) 2021 Jan Gnip
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_HX711_H_
+#define ZEPHYR_DRIVERS_SENSOR_HX711_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <device.h>
+#include <kernel.h>
+#include <drivers/sensor.h>
+#include <drivers/gpio.h>
+
+#include <drivers/sensor/hx711.h>
+
+struct hx711_data {
+	const struct device *dout_gpio;
+	const struct device *sck_gpio;
+	const struct device *rate_gpio;
```

Comment:
this is not required when using `gpio_dt_spec`

Reply:
Done.